kihuha/iReporter-db
# #163431613 - Configure persistent image storage

#### What does this PR do?
Configures persistent storage for images

#### Description of the task to be completed
- Include a cloudinary addon
- Configure cloudinary for image storage on the backend

#### How should this be manually tested
Navigate to `https://kihuha.github.io/iReporter/ui/login.html` and login with your credentials. On the dashboard page, click on `new record button` and create a new record with an image upload. On the main dashboard, click on `view` to view the details of an individual record and the uploaded image should be viewed on the page.

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/163431613
